crates/store/src/db/migrations/2025062000000_setup/up.sql

```diff
@@ -34,6 +33,12 @@ CREATE TABLE accounts (
 ) WITHOUT ROWID;
 
 CREATE INDEX idx_accounts_network_prefix ON accounts(network_account_id_prefix) WHERE network_account_id_prefix IS NOT NULL;
+CREATE INDEX idx_accounts_id_block ON accounts(account_id, block_num DESC);
```

Comment:
I didn't know you could order the index. Does this determine the search order priority? So it will scan the index descendingly?